### PR TITLE
Fix outdated user workload monitoring configuration

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

**Review by haasad — APPROVED** [review]
LGTM

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 1]
```diff
@@ -102,16 +101,38 @@ This table shows the monitoring components you can configure and the keys used t
...
+default::
++
+[source,yaml]
+----
+prometheusOperator: {}
+prometheus: ${openshift4_monitoring:configs:prometheusK8s}
+thanosRuler: {}
+----
+
+A dictionary holding the configurations for the https://docs.openshift.com/container-platform/latest/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#configurable-monitoring-components_configuring-the-monitoring-stack[user workload monitoring components].
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc:127]:
```suggestion
A dictionary holding the configurations for the https://docs.openshift.com/container-platform/latest/monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack_configuring-the-monitoring-stack[user workload monitoring components].
```

**Review by simu — APPROVED** [review]
LGTM. Looks like the link you copy-pasted is broken, might be worth it to fix it in the original location as well (unfortunately that location is not close enough to the diff to make a suggestion).

File: docs/modules/ROOT/pages/references/parameters.adoc (hunk 1 above)
Comment (DebakelOrakel) [reply docs/modules/ROOT/pages/references/parameters.adoc]: ah shoot, didn't check that :D

**Pull request merged**